assets/src/application/ui/input/InputWithLabel/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { InputWithLabel as InputWithLabelAdapter } from '@infraUI/inputs';
+import './style.scss';
+
+interface InputWithLabel {
+	label: React.ReactNode;
+	labelPosition: 'left' | 'right';
+}
+
+const InputWithLabel: React.FC<InputWithLabel> = ({ children, label, labelPosition, ...props }) => {
+	const leftLabel = labelPosition === 'left' && label;
+	const rightLabel = labelPosition === 'right' && label;
```

Comment:
May be set a default value for `labelPosition`?

Reply:
yes, good one, let's have right side as default